packages/app-builder/src/routes/ressources+/settings+/data-model+/tables+/$tableId+/exported-fields.ts

```diff
@@ -0,0 +1,150 @@
+import { isAdmin } from '@app-builder/models/user';
+import { initServerServices } from '@app-builder/services/init.server';
+import { getRoute } from '@app-builder/utils/routes';
+import { type ActionFunctionArgs, redirect } from '@remix-run/node';
+import z from 'zod';
+
+const ingestedDataFieldSchema = z.object({
+  path: z.array(z.string()),
+  name: z.string(),
+});
```

Comment:
Isn't it already defined in : packages/app-builder/src/queries/settings/scenarios/schema.ts ?